### ISSUE TITLE
Paths with env variables, e.g. %USERPROFILE%\go\bin

### ISSUE THREAD

**Issue by iva2k** [rix0rrr/WindowsPathEditor#33]
Paths with env variables, e.g. %USERPROFILE%\go\bin are not working. Apparently, env variables are not being substituted.
However, it is very common practice to include environment variables for many different reasons. Would be nice to support it.

**Comment by RockyMM** [comment]
@iva2k could you describe the issue a bit? What exactly is not working for you?

**Comment by iva2k** [comment]
wow, 3 years later...

I want to specify the path as `%USERPROFILE%\go\bin`, and keep it this way in the path environment, without substitution. So that the substitution will be done by Windows when using the variable.

**Comment by RockyMM** [comment]
Yeah, sorry 😁  I was wondering is it something I was having a problem myself.

I will try out what you described out of curiosity. I would expect that WindowsPathEditor has nothing with the env substitution in the paths.

**Comment by RockyMM** [comment]
I just tried this out. In the Add Dir dialog in the Folder input I pasted a path with a variable, e.g. `%USERPROFILE%\Documents\My Downloads\windowspatheditor-1.7`. What WindowsPathEditor did it replaced `%USERPROFILE%` with `%HOME%`. But it did apply it to my user path.

So I would say it works.
